### DISCUSSION TITLE
Graphql Inspector in the Github Actions

### DISCUSSION THREAD

**Discussion by mfrancaleal** [community/community#25006] [Actions]
<p>Hello guys… I’m a noob with Graphql Inspector… I’m trying to use in the GitHub Actions and show me this error:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/6/36778ac1f0ab46c9ec6c06257a03b41734d4a158.png" data-download-href="/uploads/short-url/7LPTRsEHv8CUktw2sJAC9AAAVvi.png?dl=1" title="Grapql" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/8262328/181088136-2750952d-4f21-4591-86a6-6fb087c0f598.png" alt="Grapql" data-base62-sha1="7LPTRsEHv8CUktw2sJAC9AAAVvi" width="690" height="432" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/6/36778ac1f0ab46c9ec6c06257a03b41734d4a158_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Grapql</span><span class="informations">972×609 28.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Can someone help me?</p>

**Comment by rowi1de** [comment]
<p>I’ve the same problem with this action, wasn’t able to figure out what’s wrong.<br>
You might want to post your workflow in this issue as well: <a href="https://github.com/kamilkisiela/graphql-inspector/issues/1488" class="inline-onebox" rel="noopener nofollow ugc">GitHub Actions: schema.graphql in subdirectory · Issue #1488 · kamilkisiela/graphql-inspector · GitHub</a></p>

**Comment by mfrancaleal** [comment]
<p>Thankyou <a class="mention" href="/rowi1de">@rowi1de</a> . I posted my problem. I hope someone ca help me.</p>

**Comment by mfrancaleal** [comment, marked as answer]
<p>A silly detail. The name of my branch is <strong>main</strong> not <strong>master</strong>. I changed and it worked. Tks.</p>
